src/main/java/org/jenkinsci/plugins/tokenmacro/impl/BuildLogMacro.java

```diff
@@ -24,6 +24,7 @@
     public static final String MACRO_NAME = "BUILD_LOG";
 
     public static final int MAX_LINES_DEFAULT_VALUE = 250;
+    public static final int MAX_LINE_LENGTH_DEFAULT_VALUE = 0;

```

Comment:
I'm wondering if setting the default value to `-1` would make more sense. `0` may still be a valid value for `maxLineLength`?

Reply:
While `0` is valid for a length, it is meaningless in this context. Besides, it is better to be consistent with other similar params as this gets exposed to end users.